tests/ltp/patches/fgetxattr03.patch

```diff
@@ -0,0 +1,40 @@
++ Currently xattr is not enabled while mounting root file system. Patch is
++ to mount root file system with xattr enabled and then use it for the test.
+diff --git a/testcases/kernel/syscalls/fgetxattr/fgetxattr03.c b/testcases/kernel/syscalls/fgetxattr/fgetxattr03.c
+index d293ffca9..66394be3b 100644
+--- a/testcases/kernel/syscalls/fgetxattr/fgetxattr03.c
++++ b/testcases/kernel/syscalls/fgetxattr/fgetxattr03.c
+@@ -29,7 +29,11 @@
+ #define XATTR_TEST_KEY "user.testkey"
+ #define XATTR_TEST_VALUE "this is a test value"
+ #define XATTR_TEST_VALUE_SIZE 20
+-#define FILENAME "fgetxattr03testfile"
++#define FILENAME MNTPOINT"/fgetxattr03testfile"
```

Comment:
Will that really work? Please change to
#define FILENAME "mntpoint/fgetxattr03testfile" 
like you used in below files.

Reply:
Yes. But I have updated as requested.